--- ext/-test-/printf/printf.c	2014-11-03 21:03:50.000000000 -0800
+++ ../new/ext/-test-/printf/printf.c	2015-04-14 17:30:20.808749300 -0700
@@ -20,7 +20,7 @@
 }
 
 static char *
-utoa(char *p, char *e, unsigned int x)
+uutoa(char *p, char *e, unsigned int x)
 {
     char *e0 = e;
     if (e <= p) return p;
@@ -71,12 +71,12 @@
 	    *p++ = '0';
 	}
 	if (!NIL_P(v = rb_hash_aref(opt, ID2SYM(rb_intern("width"))))) {
-	    p = utoa(p, format + sizeof(format), NUM2UINT(v));
+	    p = uutoa(p, format + sizeof(format), NUM2UINT(v));
 	}
 	if (!NIL_P(v = rb_hash_aref(opt, ID2SYM(rb_intern("prec"))))) {
 	    *p++ = '.';
 	    if (FIXNUM_P(v))
-		p = utoa(p, format + sizeof(format), NUM2UINT(v));
+		p = uutoa(p, format + sizeof(format), NUM2UINT(v));
 	}
     }
     *p++ = cnv;
